gallery/src/components/NxDrawer/NxDrawerPage.tsx

```diff
@@ -326,6 +327,35 @@ export default function NxDrawerPage() {
         </NxTile.Content>
       </NxTile>
 
+      <NxTile>
+        <NxTile.Header>
+          <NxTile.HeaderTitle>
+            <NxH2>NxDrawer With NxToast Example</NxH2>
+          </NxTile.HeaderTitle>
+        </NxTile.Header>
+        <NxTile.Content>
+          <NxP>
+            An example of <NxCode>NxDrawer</NxCode> with <NxCode>NxToast</NxCode>. If a toast is already open and
+            then a drawer is opened the toast appears on top of the drawer. This is the same if the toast is opened
+            from a button inside the drawer. Both the drawer and toast will close if a toast is closed.
```

Comment:
> Both the drawer and toast will close if a toast is closed.

That doesn't sound desirable

Reply:
> That doesn't sound desirable

Nor does it seem to be accurate as I try it out

Reply:
> > That doesn't sound desirable
> 
> Nor does it seem to be accurate as I try it out

I just tried it out and it does happen for me. If a drawer and toast are both present, when I close a toast, it closes the drawer too. Does that happen for you as well?  I assume this has something to do with the layout of NxDrawer, when you click outside of the drawer, it automatically closes it?

Reply:
> I just tried it out and it does happen for me. If a drawer and toast are both present, when I close a toast, it closes the drawer too. Does that happen for you as well?

Hmm it does now; I thought it didn't yesterday.  You're right that it's not actually about closing the toast, it's about clicking on the toast at all.  I think that bears checking with Design.  Since the toasts are in front of the drawer, it doesn't feel intuitive that clicking on them would close the drawer.

Reply:
> > I just tried it out and it does happen for me. If a drawer and toast are both present, when I close a toast, it closes the drawer too. Does that happen for you as well?
> 
> Hmm it does now; I thought it didn't yesterday. You're right that it's not actually about closing the toast, it's about clicking on the toast at all. I think that bears checking with Design. Since the toasts are in front of the drawer, it doesn't feel intuitive that clicking on them would close the drawer.

Checked with design and decided to make an exception to not close the drawer when toasts are interacted with: 58b5e5ccbbb3a0f40853646e14d9039260c70626